<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPersonalAccessTokenManagerGuiComp</class>
 <widget class="QWidget" name="CPersonalAccessTokenManagerGuiComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Personal Access Token Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="UserIdLabel">
     <property name="text">
      <string>User: &lt;not set&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="MainSplitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="LeftPanel">
      <layout class="QVBoxLayout" name="leftPanelLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TokenListLabel">
         <property name="text">
          <string>Personal Access Tokens</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font-weight: bold;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="TokenListWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="CreateTokenButton">
           <property name="text">
            <string>Create</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RevokeTokenButton">
           <property name="text">
            <string>Revoke</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DeleteTokenButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RefreshButton">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RightPanel">
      <layout class="QVBoxLayout" name="rightPanelLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TokenDetailsLabel">
         <property name="text">
          <string>Token Details</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font-weight: bold;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="TokenDetailsText">
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
